### ISSUE TITLE
helpers around management of non-fungible tokens and their token mappings

### ISSUE THREAD

**Issue by hathbanger** [versatus/versatus-javascript#65]
basically, we need to start building _guardrails helpers_ around the data field in a non-fungible program. The data field is arbitrary data but if we provide them w tooks that can make it easy for them to manage structured data in the data field, it will help developers out immensely with working with data that is specific to tokenIds.

have some good stuff in this commit here: https://github.com/versatus/versatus-javascript/pull/62/commits/dca7fc38c78cd03a62c31341cb39be77f6610ee3

**Comment by hathbanger** [comment]
closing this as i may have been overthinking how we approach data mgmt. [THIS ISSUE helps clarify my new direction here.](https://github.com/versatus/lasr/issues/111). 

TLDR: avoid needing to deal with too much nesting in the data field. i've started to using more verbose keys i.e. `tweet-<iso_date>` or `pokemon-<token_id>`
